packages/smithy-aws-core/src/smithy_aws_core/credentials_resolvers/static.py

```diff
@@ -0,0 +1,19 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  SPDX-License-Identifier: Apache-2.0
+from smithy_aws_core.identity import AWSCredentialsIdentity
+from smithy_core.aio.interfaces.identity import IdentityResolver
+from smithy_core.interfaces.identity import IdentityProperties
+
+
+class StaticCredentialsResolver(
+    IdentityResolver[AWSCredentialsIdentity, IdentityProperties]
```

Comment:
Did changing `AWSCredentialsResolver` to a `type` make it unusable here? Wondering at the change.

Reply:
Yeah - type aliases can't be used as base classes, which makes sense.  I think the type alias is still useful though.